src/AppBundle/Admin/RoleAdmin.php

```diff
@@ -41,8 +41,8 @@ protected function configureShowFields(ShowMapper $showMapper)
     protected function configureFormFields(FormMapper $formMapper)
     {
         $formMapper
-            ->add('title')
-            ->add('description')
+            ->add('title', null, ['required' => false])
```

Comment:
I did this, because if add empty role in Admin, then decide to delete it, it will say  'You must fill this filled'... not very friendly... you have to fill in something to this field, only then you can delete it)) Thats why I made it not required... it is Ok?